plugins/esi/combo_handler.cc

```diff
@@ -323,8 +333,38 @@ TSPluginInit(int argc, const char *argv[])
     return;
   }
 
-  if ((argc > 1) && (strcmp(argv[1], "-") != 0)) {
-    COMBO_HANDLER_PATH = argv[1];
+  if (argc > 1) {
+    int c;
+    static const struct option longopts[] = {
+      {"max-files", required_argument, nullptr, 'f'}, {nullptr, 0, nullptr, 0},
+    };
+
+    int longindex = 0;
+    optind        = 1; // Force restart to avoid problems with other plugins.
```

Comment:
i thought we dont need this after TS-3245 ?

Reply:
Possibly, but why not be safe? I can take it out.

Reply:
Probably just to be consistent with all others, IMHO.